Package.swift

```diff
@@ -30,7 +30,8 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(url: "https://github.com/Esri/arcgis-maps-sdk-swift", .upToNextMinor(from: "200.3.0"))
+        .package(url: "https://github.com/Esri/arcgis-maps-sdk-swift", .upToNextMinor(from: "200.3.0")),
+        .package(url: "https://github.com/Esri/Dejavu", branch: "main")
```

Comment:
Are we okay pointing at main for now? Do we need to come up with a tagging plan for Dejavu?

Reply:
I've reached out to GH admin for advice on this.

Reply:
@rolson any preference on a tagging plan for Dejavu? `v1.0`/`v200.x`?

Reply:
I think it can be v1